network/base/views.py

```diff
@@ -306,10 +308,79 @@ def station_view(request, id):
     form = StationForm(instance=station)
     antennas = Antenna.objects.all()
 
+    try:
+        satellites = Satellite.objects.filter(transmitters__alive=True).distinct()
+    except:
+        pass  # we won't have any next passes to display
```

Comment:
I think `filter()` fails silently when empty, but this try/except seems weird to me (see bellow).